noise_raster/raster_processing.py

```diff
@@ -66,20 +56,68 @@ def sum_sound_level_3D(sound_levels: np.array):
     if len(sound_levels.shape) != 3:
         raise ValueError('Input array not 3D ')
 
-    sound_levels_fl32 = sound_levels.astype(np.float32)
+    sound_levels.astype(np.float32)
 
-    l, m, n = sound_levels_fl32.shape
+    l, m, n = sound_levels.shape
 
     sound_pressures = np.zeros((l, m, n)).astype(np.float32)
     sum_pressures = np.zeros((l, m, n)).astype(np.float32)
     out = np.zeros((l, m, n)).astype(np.float32)
 
-    sound_pressures = np.power(10, 0.1 * sound_levels_fl32)
+    sound_pressures = np.power(10, 0.1 * sound_levels)
     sum_pressures = np.sum(sound_pressures, axis=0)
-    out = 10 * np.log10(sum_pressures)
-    rounded_out = out.round(decimals=1)
+    out = (10 * np.log10(sum_pressures)).round(decimals=1)
+
+    return out
+
+def create_temp_directory():
+    """
+    Create a sub-directory in the current temp folder to hold intermediate files.
+    """
+    # Get current date and time
```

Comment:
Remark on code comments (not a blocker for merging):

This comment is superfluous as the the following code line is trivial and self-explaining. It's generally preferrable to add comments only in case it needs to be explained _why_ a certain code block is needed and not _what_ is happening in the code block.

Reply:
I removed this.